beacon-chain/core/execution/upgrade.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // UpgradeToBellatrix updates inputs a generic state to return the version Bellatrix state.
 // It inserts an empty `ExecutionPayloadHeader` into the state.
-func UpgradeToBellatrix(ctx context.Context, state state.BeaconState) (state.BeaconState, error) {
+func UpgradeToBellatrix(_ context.Context, state state.BeaconState) (state.BeaconState, error) {
```

Comment:
Are we sure about this change? It hides an error that is unrelated to this PR and may make it harder to debug later if the function is not properly implemented. The parameter should go, or just keep the error. Incidentally. Each time you push you make stale my previous approval. The PR does look good to me

Reply:
@potuz I don't have an opinion about this, I just did it so "Deepsource" wouldn't protest. If we can merge it with the "Deepsource" error, I can revert it, no problem.

Are you suggesting to remove the parameter all the way? Or merging ignoring "Deepsource"?

Reply:
I would suggest to leave the error as is since this is not the scope of the PR, we can wait until Monday and let @terencechain chime in, since he'll write the PR implementing the upgrade, but either way, hiding the error i think shouldn't be the best option

Reply:
+1

Let's leave the error as it is

Reply:
ok